### PR TITLE
Tai/documentation

### PR DESCRIPTION
**Pull request by ctlai95**

### CODE REVIEW

**Comment by henrytran28** [comment]
Can you split this PR into documentation and tuples please?

**Comment by ctlai95** [comment]
haha i was hoping you wouldn't notice 😨

File: src/point/point.py [hunk 1]
```diff
@@ -4,14 +4,41 @@
...
-    """A point is a coordinate on the board with an x and y position"""
+    """A point is a coordinate on the board with an x and y position."""
 
     def __init__(self, x, y):
+        """
+        Initialize a Point object.
+
+        Args:
+            x (int): The x coordinate
+            y (int): The y coorindate
```
Comment (henrytran28) [inline src/point/point.py]: periods

File: src/board/board.py [hunk 2]
```diff
@@ -55,7 +65,13 @@ def update_matrices(self):
             self.fill_matrix(self.current_tetromino_matrix, square)
 
     def get_ghost_tetromino(self):
-        """Returns a gray clone of the current tetromino and moves it down by the maximum amount"""
+        """
+        Return a gray clone of the current tetromino and moves it down by the maximum amount.
+
+        Returns:
+            Tetromino: The ghost tetromino
```
Comment (henrytran28) [inline src/board/board.py]: period

File: src/keyboard/keyboard.py [hunk 3]
```diff
@@ -1,45 +1,42 @@
...
+    def __init__(self, window, board, movement):
+        """
+        Initialize a Keyboard object.
 
-class Keyboard:
-    def __init__(self, window, board, movement, key_handler):
+        Args:
+            window (Window): The game's window object
+            board (Board): The game's board object
+            movement (Movement): The game's movement handler
```
Comment (henrytran28) [inline src/keyboard/keyboard.py]: Periods

**Pull request merged**